openquake/hazardlib/sourcewriter.py

```diff
@@ -442,6 +442,12 @@ def get_source_attributes(source):
             for data in source.data:
                 weights.append(data[0].weight)
             attrs['rup_weights'] = numpy.array(weights)
+    elif isinstance(source, PointSource):
+        tom = source.temporal_occurrence_model
+        if isinstance(tom, NegativeBinomialTOM):
+            attrs['tom'] = 'NegativeBinomialTOM'
+            attrs['mu'] = tom.mu
+            attrs['alpha'] = tom.alpha
```

Comment:
Not the right place for this, since the tom instance should not be an attribute of PointSource, only of SourceGroup.